native/packages/react-native-backpack/libraries/Badge/ios/react/classes/BPKBadgeWrapper.m

```diff
@@ -0,0 +1,104 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKBadgeWrapper.h"
+
+#import <Backpack2/Badge.h>
+
+#import <React/RCTUIManagerUtils.h>
+#import <React/RCTUIManager.h>
+#import <React/RCTShadowView.h>
+#import <React/RCTComponent.h>
+//#import <React/UIView+React.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@interface BPKBadgeWrapper() <RCTComponent>
+@property (nonatomic, strong) BPKBadge *inner;
+@property (nonatomic, weak) RCTBridge *bridge;
+
+// Message
+- (void)setMessage:(nullable NSString *)message;
+- (nullable NSString *)message;
+
+// Badge Type
+- (void)setType:(BPKBadgeType)type;
+
+- (void)setup;
+@end
+
+@implementation BPKBadgeWrapper
+
+- (instancetype)initWithBridge:(RCTBridge *)bridge {
```

Comment:
Is this the RN equivalent of `initWithFrame` and `initWithCoder`?

Reply:
No, we just need a reference to the bridge for this view